project/plugins.sbt

```diff
@@ -1,19 +1,21 @@
 resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 
-resolvers += "Typesafe Releases" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
 resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.16.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.19.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "x.x.x")
```

Comment:
Can this be removed? its already declared below with a version

Reply:
Removed it. Thanks.